### ISSUE TITLE
verify no changes needed to flask session for samesite cookie policy change

### ISSUE THREAD

**Issue by balmas** [alpheios-project/alpheios_nemo_ui#307]
see https://blog.chromium.org/2019/10/developers-get-ready-for-new.html

**Comment by balmas** [comment]
I don't believe any flask changes are required but we do need to update google analytics and auth0 cookie handling code.

For google analytics the solution at https://www.simoahava.com/analytics/cookieflags-field-google-analytics/ appears to work for now:

```
 ga('create', 'UA-XXXXX-Y', {
  cookieFlags: 'max-age=7200;secure;samesite=none'
});
```

**Comment by balmas** [comment]
the external cookies (google analytics and auth0) are addressed but to made this adjustment to the flask session cookie we need to upgrade Flask and its libraries so that we can get the fix in pallets/werkzeug#1549

**Comment by balmas** [comment]
It should only be an issue if we wanted to support the texts site being run in an iframe from another domain though, so I think it's not critical, although there might be other reasons to upgrade Flask.  Will enter a new issue for us to consider at some point.
